From dda42b80b56bf8136049bccbab2648a573a48551 Mon Sep 17 00:00:00 2001
From: William Woodall <wjwwood@gmail.com>
Date: Tue, 12 Mar 2013 11:15:36 -0700
Subject: [PATCH] Install package.xml

---
 CMakeLists.txt |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 129b83d..f3c4d84 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -301,6 +301,9 @@ else (build_errors)
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/pkgconfig/${pkgconfig}.pc DESTINATION ${LIB_INSTALL_DIR}/pkgconfig COMPONENT pkgconfig)
   endforeach()
 
+  # Install catkin package.xml
+  install(FILES package.xml DESTINATION share/gazebo)
+
   ########################################
   # If present, load platform-specific build hooks.  This system is used,
   # for example, by the Ubuntu overlay (in the gazebo-release repo), to
-- 
1.7.9.5

